src/components/organisms/RadioModal/RadioModal.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect } from "react";
+import "./RadioModal.scss";
+
+const sound = new Audio(
+  "https://www.soundhelix.com/examples/mp3/SoundHelix-Song-15.mp3"
```

Comment:
2 requests
1. Can this please come from the playa venue configuration? Can be passed in via props if you like.
2. Can the venue configuration we use be an array, as we may want to do a small bit of UI during burn and allow multiple radio stations.